### DISCUSSION TITLE
生产环境broker版本从4.5.2升级至4.9.6后硬盘使用率升高一倍

### DISCUSSION THREAD

**Discussion by R1ghtHereWaiting** [apache/rocketmq#7215] [Q&A]
<img width="818" alt="image" src="https://github.com/apache/rocketmq/assets/83198160/aa8ade86-5e0a-4b55-a8e3-e5f7ce215982">
<img width="833" alt="image" src="https://github.com/apache/rocketmq/assets/83198160/c1801174-fc75-4056-968f-322c8cb3c559">
以上为生产环境Grafana的监控指标
未升级时磁盘每天上涨一部分空间之后，就会掉空间释放使用率掉下去一部分
升级后使用率只增不减

**Comment by onejimmyboy** [comment]
定时删除过期文件不生效？可以看看store.log的日志
